### PR TITLE
compat

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: macs.php [hunk 1]
```diff
@@ -0,0 +1 @@
+i<?php include 'data.php';  ?>
```
Comment (JeroenDeDauw) [inline macs.php]: The moment when you realize there is actually a typo here

Comment (sandb) [inline macs.php]: Hehe - will fix

Comment (sandb) [inline macs.php]: Thx! Fixed. You using it somewhere?

\g
sandb

Comment (JeroenDeDauw) [inline macs.php]: Nope, but whoever carroarmato0 is on the 0x20 irc is :)

Comment (sandb) [inline macs.php]: Hehe, ok :-) wow you still tracking #0x20? Been a long time for me...

See you on 32c3?

Greets,
Sandb

Comment (JeroenDeDauw) [inline macs.php]: GitHub PRs - the ideal place to catch up with old friends :)

Not sure if I've ever removed a channel from my IRC client... And yeah, see you at the ccc!

**Pull request merged**